docs/ref/forms/models.txt

```diff
@@ -1,15 +1,275 @@
-====================
-Model Form Functions
-====================
-
-Model Form API reference. For introductory material about model forms, see the
-:doc:`/topics/forms/modelforms` topic guide.
+================================================
+ModelForm ``Meta`` Options and Factory Functions
+================================================
 
 .. module:: django.forms.models
-   :synopsis: Django's functions for building model forms and formsets.
+    :synopsis: ModelForm API reference for inner ``Meta`` class and factory
+        functions
+
+.. admonition:: Other ModelForm-related documentation
+
+    This document is an API reference for the :class:`~django.forms.ModelForm`
+    class. For examples or additional context, please refer to the
+    documentation topic :doc:`Creating forms from models
+    </topics/forms/modelforms>`.
+
+.. _ref-modelform-meta-options:
+
+ModelForm ``Meta`` Options
+==========================
+
+.. currentmodule:: django.forms
+
+The :class:`~django.forms.ModelForm` helper class can be used to build forms
+that closely reflect specific Django models. The exact structure of the form
+produced can be controlled by setting metadata options as attributes of an inner
+``Meta`` class.
+
+::
+
+    # Example using some sort of Book model
+    
+    from django.forms import ModelForm
+    from myLibraryApp.models import Book
+
+    class BookForm(ModelForm)
+        class Meta:
+            model = Book
+            fields = [ 'title', 'author' ]
+            help_texts = {
+                'title' : 'The title of the book',
+                'author': 'The author of the book'
+            }
+            # ... other attributes
+
+.. admonition:: Invalid Meta class attributes
+
+    While any number of attributes can be included in a ``Meta``
+    class, **only attributes that match those described below will be used**.
+    When troubleshooting or encountering unexpected behavior, be sure to check
+    for typos or misnamed attributes because these will **not**, in and of
+    themselves, raise an error (though they might cause another error to be
+    raised if they affect a required attribute).
+
+It is **not** necessary for every option to be set for each ``Meta`` class,
+though some (indicated below) are required.
+
+In most cases, if an option isn't set, or if a model's field isn't used with a
+particular option, Django will fallback on whatever has already been defined for
+a field in their model or whatever defaults might exist for that field's type.
+
+.. _ref-modelform-meta-required:
+
+Required ``Meta`` Options
```

Comment:
No need to group the attributes this way, since attributes specify whether they're optional or required.

Reply:
Yep-- I've reordered them to be purely alphabetically now.